docker/docker_linux.go

```diff
@@ -120,6 +172,26 @@ func CgroupCPUDocker(containerid string) (*cpu.TimesStat, error) {
 	return CgroupCPU(containerid, common.HostSys("fs/cgroup/cpuacct/docker"))
 }
 
+// CgroupPID retrieves the PIDs running within a given container.
+func CgroupPID(containerID string, base string) ([]string, error) {
```

Comment:
maybe return `[]int32` instead since this is pids? also should we call it CgroupPIDs (plural) for clarity of what you're getting back?